src/app/backend/apihandler.go

```diff
@@ -18,17 +18,35 @@ import (
 	"log"
 	"net/http"
 	"strconv"
+	"strings"
 
 	restful "github.com/emicklei/go-restful"
 	client "k8s.io/kubernetes/pkg/client/unversioned"
 )
 
+func WsLogger() restful.FilterFunction {
```

Comment:
I've already added logging of JSON payload of the request, but for the response, which is more important, it's a little bit more complicated. It seems, that we don't have access to response content by `restful.Response` object and to change it, we would need to override/extend it somehow. Do you see any solution for this? Will it be acceptable to perform such modifications?